.detekt.yml

```diff
@@ -0,0 +1,584 @@
+build:
```

Comment:
can you clarify what's different to the app config here?

Reply:
Done